packages/mdc-layout-grid/_mixins.scss

```diff
@@ -48,6 +48,9 @@
 
   @supports (display: grid) {
     width: auto;
+    max-width: 100%;
+    max-height: 100%;
+    overflow: hidden;
```

Comment:
That makes sense. What worries me about using `max-` width/height for this as a fix is that I can't see anyone being happy with their image being stretched. What if we use `min-` width/height? That way the aspect ratio of the image is preserved, even if the crop and resolution is unfavorable.

Would like to get @traviskaufman and @yeelan0319 to weigh in. 

Looks good so far though, thanks!